packages/react-core/src/components/Skeleton/Skeleton.tsx

```diff
@@ -21,6 +23,7 @@ export const Skeleton: React.FunctionComponent<SkeletonProps> = ({
   height,
   fontSize,
   shape,
+  screenreaderText = 'Loading...',
```

Comment:
I understand accessible defaults, but if this should NOT be the default whenever there exists more than 1 skeleton we might want to use a React context to conditionally add the text rather than ask the consumer to pass an empty string for every Skeleton after the first.

Is this your recommended usage?

Reply:
Yeah, I agree with you, I was thinking along those lines myself. I think based on our conversation, it makes more sense to avoid the complexity and just remove the default, allowing consumers to intentionally add text instead.